Makefile

```diff
@@ -35,16 +35,14 @@ NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -exec $(STAT) {} \; \
                    | sort -r | head -n 1 | sed "s/.* //")
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
 GO_VERSION     = 1.7.3
-GO_BUILD       = go build -i -v -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION)"
+GO_BUILD       = env GOOS=linux GOARCH=amd64 go build -i -v -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION)"
```

Comment:
We do. If this is run in a container, Docker doesn't like when the command you pass it leads off with ENV=val. Using `env` solves that problem.